README.md

```diff
@@ -0,0 +1,288 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
```

Comment:
You have done great by using the Microverse readme template. For more professional readme documentation, please make sure to do the following:

- customize this template making sure to have all main headers from the [README template](https://github.com/microverseinc/readme-template) present. The only optional section is FAQ.
- make sure to have added the section Future Features, including 1-3 potential features to be added.
- remove the section descriptions, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
- remove or replace the Microverse logo